analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -58,7 +58,7 @@ import org.ossreviewtoolkit.utils.ort.Environment
  * The class to run the analysis. The signatures of public functions in this class define the library API.
  */
 class Analyzer(private val config: AnalyzerConfiguration, private val labels: Map<String, String> = emptyMap()) {
-    companion object : Logging
+    internal companion object : Logging
```

Comment:
Would it be reasonable to use `internal` is minimum scope, to avoid switching it back an forth?

Reply:
What do you mean by "switching it back an forth"? Do you mean in case the `private` entries would need to become `internal` for some reason in the future?

Reply:
As we already have several `private companion object : Logging` we could do a bulk change to `internal companion object : Logging` any time later, if needed.